src/Misc/layoutbin/powershell/Add-XamarinAndroidCapabilities.ps1

```diff
@@ -1,4 +1,22 @@
 [CmdletBinding()]
 param()
+$xamarinAndroidVersion = [Version]'0.0.0.0'
 
-$null = Add-CapabilityFromRegistry -Name 'Xamarin.Android' -Hive 'LocalMachine' -View 'Registry32' -KeyName 'Software\Novell\Mono for Android' -ValueName 'InstalledVersion'
+# check VS2017 setup packages for latest Xamarin Android version
+$vs2017Instances = Get-VSSetupInstance -All | Select-VSSetupInstance -Require 'Xamarin.Android.Sdk'
```

Comment:
Requires https://github.com/Microsoft/vssetup.powershell to be installed. Following up on the best way to include this on the agent.